src/word/hlxMediaHandler.js

```diff
@@ -0,0 +1,224 @@
+/*
```

Comment:
I would probably call this file `aemMediaHandler.js`.

Reply:
Sure, no problem, I had named it like this because the repository was still called `helix-mediahandler`.